.bumpversion.cfg

```diff
@@ -0,0 +1,11 @@
+[bumpversion]
+current_version = 0.0.0
```

Comment:
We need to make sure we don't use a 1.x version for the core package until we know it's stable

Reply:
Created an initial tag: `v0.0.0` and ran semantic locally:
```
# 0.1.0 (https://github.com/IBM/python-sdk-core/compare/v0.0.0...v0.1.0) (2019-03-13)

### Features

    * core: create a separate core for IBM python SDK (2421459 (https://github.com/IBM/python-sdk-core/commit/2421459))
```